integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -95,6 +95,7 @@ else
                 "k8s-shared-volume.bats" \
                 "k8s-volume.bats" \
                 "k8s-nginx-connectivity.bats" \
+				"k8s-hugepages.bats" \
```

Comment:
It seems there is an indent issue, the new line is not aligned with others.

Reply:
I just used an extension to fix the format in this file